src/codemagic/__version__.py

```diff
@@ -1,5 +1,5 @@
 __title__ = 'codemagic-cli-tools'
 __description__ = 'CLI tools used in Codemagic builds'
-__version__ = '0.37.1.dev'
+__version__ = '0.38.0'
```

Comment:
It is better to keep the `.dev` suffix. Otherwise it is difficult to distinguish development version from proper release.
```suggestion
__version__ = '0.38.0.dev'
```